waw/src/lib.rs

```diff
@@ -3,7 +3,9 @@
 #![doc = include_str!("../../README.md")]
 #![warn(missing_docs)]
 
+
 /// Audio buffer structs
+
```

Comment:
don't need to change this part of the code,
in the rust project,  there is no need to give the br in this part

Reply:
yes okay
i will change this

Reply:
okay
this is solved